std/algorithm/setops.d

```diff
@@ -853,9 +857,11 @@ NWayUnion!(less, RangeOfRanges) nWayUnion
     auto witness = [
         1, 1, 1, 4, 4, 7, 7, 7, 7, 8, 8
     ];
-    assert(equal(nWayUnion(a), witness));
+    assert(equal(multiwayMerge(a), witness));
 }
 
+alias nWayUnion = multiwayMerge;
```

Comment:
cool, also please add `alias NWayUnion = NWayMerge` and mention it in the doc. Forgot to mention that.